hoot-services/src/main/java/hoot/services/controllers/grail/PullApiCommand.java

```diff
@@ -107,7 +107,8 @@ private void getApiData() {
             double maxBboxArea = params.getMaxBBoxSize();
 
             if (bboxArea > maxBboxArea) {
```

Comment:
Do we maybe want to remove this if statement in general?

https://radiantsolutions.atlassian.net/browse/VGI-1852

Reply:
Are these bbox limits actually for the OSM API and not the Overpass API?

Reply:
was being used by both due to us setting a default of 0.25 in GrailParams.java